Backend/Remora.Discord.Gateway/DiscordGatewayClient.cs

```diff
@@ -1047,6 +1047,16 @@ private async Task<Result> GatewayReceiverAsync(TimeSpan heartbeatInterval, Canc
 
                 var receivedAt = DateTime.UtcNow;
 
+                if (receivedPayload.Error is Results.WebSocketError { State: WebSocketState.Aborted })
+                {
+                    // request reconnect.
+                    _shouldReconnect = true;
+
+                    // idk if connection is resumable.
+                    _isSessionResumable = false;
```

Comment:
I am not sure if it should be resumable so I set it to false in case to start a completely new session.